test/test_proxy_tensor.py

```diff
@@ -1259,7 +1259,6 @@ def f(a, b, c, d, e):
     xfail('nn.functional.cross_entropy', ''),  # aten.size.default - couldn't find symbolic meta function/decomposition
     xfail('nn.functional.ctc_loss'),  # aten._ctc_loss.Tensor - couldn't find symbolic meta function/decomposition
     xfail('nn.functional.embedding_bag', ''),  # aten._embedding_bag_forward_only.default - couldn't find symbolic meta fun...
-    xfail('nn.functional.embedding', ''),  # argument 'size' must be tuple of ints, but found element of type tor...
```

Comment:
`embedding_renorm_` is used in `embedding`, which is why I add the decomp. Otherwise, this fails.

Reply:
@ezyang PTAL at this whole stack. The first 3 PRs (select, embedding, and this one) are required to make this test pass, which was the original motivation.

I've also added another PR with improved error checking. But I'm not sure if that one is strictly needed for decomps (since other embedding* decomps don't have all the checks right now). If it is, please review it as well. But I'll keep that one as a draft for now to avoid pinging other people.